src/arduino-rfm/lorawan-arduino-rfm.h

```diff
@@ -91,6 +96,19 @@ class LoRaWANClass
     unsigned int getFrameCounter();
     void setFrameCounter(unsigned int FrameCounter);
 
+    // Declare public variable to be save on RTC RAM before deep sleep for ESP device
```

Comment:
Maybe everything related to the data you want to save should go in the compilation directives for the ESP32 and ESP8266 because it requires some extra memory